examples/rust/blackboard/creator.rs

```diff
@@ -11,8 +11,14 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
 use core::time::Duration;
+
+extern crate alloc;
+use alloc::boxed::Box;
+use alloc::format;
+
 use iceoryx2::prelude::*;
 use iceoryx2_bb_container::string::*;
+use iceoryx2_bb_log::println;
```

Comment:
Food for thought. I'm also not sure what would be the best approach.

On one hand, calling this just `println`, leads to not having to change other code, on the other hand, one could easily add an example without this import an use `println` from std. The CI would then complain and one would wonder why it fails since the other examples also use `println`.